modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_move_boxes.js

```diff
@@ -301,7 +313,8 @@ AUI.add(
 							else if (target === instance._leftBox) {
 								if (
 									rightBoxMaxItems === -1 ||
-									instance._rightBox.get('length') <
+									instance._rightBox.get('length') +
+										selectedOptions <=
```

Comment:
There is a bit of code duplication here. Maybe it would make sense to define the `adjecantBox` and `moveBtn` of current target, and have a single condition when it is disabled.

Reply:
Yes, yes, too little too late @markocikos !!

Just kidding, I'm doing exactly this